crates/ruff_linter/src/checkers/ast/analyze/statement.rs

```diff
@@ -366,6 +366,9 @@ pub(crate) fn statement(stmt: &Stmt, checker: &mut Checker) {
             if checker.enabled(Rule::SslWithBadDefaults) {
                 flake8_bandit::rules::ssl_with_bad_defaults(checker, function_def);
             }
+            if checker.enabled(Rule::SpuriousAsync) {
+                ruff::rules::spurious_async(checker, *is_async, name, body, stmt.range());
```

Comment:
For newer rules, we're trying to pass the entire node (like `function_def`, above), to ensure that callers can't pass in "incorrect" data. Do you mind revising the signature?

Reply:
I've made this change below.